KingfisherWebP.podspec

```
@@ -36,6 +36,6 @@ KingfisherWebP is an extension of the popular library [Kingfisher](https://githu
     'USER_HEADER_SEARCH_PATHS' => '$(inherited) $(SRCROOT)/libwebp/src'
   }
 
-  s.dependency 'Kingfisher', '~> 7.8.1'
+  s.dependency 'Kingfisher', '~> 7.8'
```

```suggestion
  s.dependency 'Kingfisher', '~> 7.9'
```

This library requires version `7.8.1` or higher due to a necessary bug fix in https://github.com/onevcat/Kingfisher/pull/2099